self_hosting_machinery/webgui/selfhost_fastapi_completions.py

```diff
@@ -423,3 +462,31 @@ async def _chat(self, post: ChatContext, request: Request, account: str = "XXX")
         self._id2ticket[ticket.id()] = ticket
         await q.put(ticket)
         return StreamingResponse(chat_streamer(ticket, self._timeout, req["created"]))
+
+    async def _embeddings(self, post: Embeddings, request: Request, account: str = "XXX"):
```

Comment:
is there openai style of embeddings call? if it is we should implement it in that way

Reply:
it is HF style, as Sergey asked